sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/NumberVerificationSample.md

```diff
@@ -0,0 +1,31 @@
+# NumberVerification verify
+
+Number verification involves 2 steps. In the first request, you receive a redirect URL that must be followed, in order to get teh `ApcCode`. Then in the second request, the `ApcCode` gets sent in.
+
+```C#
+using Azure.Communication.ProgrammableConnectivity;
+
+private readonly Uri _endpoint = new Uri("https://eastus.prod.apcgatewayapi.azure.com");
```

Comment:
Please use an example URL here.   We suggest using a format that makes this clear, such as:

```csharp
 Uri endpoint = new Uri("https://<< YOUR NAME >>.azure.com");
```

Reply:
I've done `https://your-endpoint-here.com` now